JavaScript/YoutubeSearch/scripts/DOM.js

```diff
@@ -0,0 +1,167 @@
+  header.appendChild(title);
+
+  const body = document.querySelector("body");
+  body.appendChild(header);
+}
+
+function searchDiv() {
+  //input tag
+  const searchBox = document.createElement("input");
+  searchBox.setAttribute("type", "text");
+  searchBox.setAttribute("id", "searchbox");
+  searchBox.setAttribute("placeholder", "search");
+
+  const searchButton = document.createElement("button");
+  searchButton.innerHTML = "&#128269;";
+
+  //search div
+  const searchDiv = document.createElement("div");
+  searchDiv.classList.add("search");
+  searchDiv.appendChild(searchBox);
+  searchDiv.appendChild(searchButton);
+
+  const body = document.querySelector("body");
+  body.appendChild(searchDiv);
+
+  //event listeners
+  searchButton.addEventListener("click", async event => {
+    query = searchBox.value;
+    await getVideos(url + query);
+    pageIndex++;
+    populateView(pageIndex);
+    const nextButton = document.querySelector(".next");
+    nextButton.style.display = "block";
+  });
+}
+
+//display div
+function displayDiv() {
+  const displayDiv = document.createElement("div");
+  displayDiv.classList.add("display");
+
+  const body = document.querySelector("body");
+  body.appendChild(displayDiv);
+}
+
+function navigationDiv() {
+  //previous button
+  const previousButton = document.createElement("button");
+  previousButton.classList.add("previous");
+  previousButton.textContent = "Previous";
+
+  //next Button
+  const nextButton = document.createElement("button");
+  nextButton.classList.add("next");
+  nextButton.textContent = "Next";
+
+  //navigation div
+  const navigationDiv = document.createElement("div");
+  navigationDiv.classList.add("navigation");
+  navigationDiv.appendChild(previousButton);
+  navigationDiv.appendChild(nextButton);
+
+  const body = document.querySelector("body");
+  body.appendChild(navigationDiv);
+
+  previousButton.addEventListener("click", event => {
+    pageIndex--;
+    populateView(pageIndex);
+  });
+
+  nextButton.addEventListener("click", event => {
+    pageIndex++;
+    populateView(pageIndex);
+  });
+}
+
+//function to fetch data and add to the global list items
+function getVideos(queryUrl) {
+  return fetch(queryUrl)
+    .then(resposne => {
+      return resposne.json();
+    })
+    .then(result => {
+      let newitems = result.items;
+      items = items.concat(newitems);
+      totalresult = totalresult + 15;
+      nextPageToken = result.nextPageToken;
+      previousPageToken = result.prevPageToken;
+    });
+}
+
+//clear the view on the page
+function clearView() {
+  const cardOneImg = document.querySelectorAll(".card");
+  const displayDiv = document.querySelector(".display");
+  for (let i = 0; i < cardOneImg.length; i++) {
+    displayDiv.removeChild(cardOneImg[i]);
+  }
+}
+
+async function populateView(pageIndex) {
+  clearView();
+  let previousButton = document.querySelector(".previous");
+  if (pageIndex <= 0) {
+    previousButton.style.display = "none";
```

Comment:
I prefer not to have any inline styling most of the time, since they are reusable again in the latter parts of the code.
i suggest you to maintain a class, and add the class into this classList.

example:
//add a css class
.none {
display: none;
}

previousButton.classList.add('none');